### ISSUE TITLE
insert_textbox fails if fontfile is pathlib.Path

### ISSUE THREAD

**Issue by cbm755** [pymupdf/PyMuPDF#1550]
Consider:
```python
fontdir = Path(...)
pdf_page.insert_textbox(..., fontfile=fontdir / ttf_file))
```
On 1.19.3 and 1.19.4, this fails with:
```
  File "/home/cbm/.local/lib/python3.10/site-packages/fitz/fitz.py", line 6569, in insert_font
    val = self._insertFont(
  File "/home/cbm/.local/lib/python3.10/site-packages/fitz/fitz.py", line 6608, in _insertFont
    return _fitz.Page__insertFont(
TypeError: in method 'Page__insertFont', argument 4 of type 'char *'
```

Of course there is an easy to workaround with `str` (but feels Python 2.x):
```python
fontdir = Path(...)
pdf_page.insert_textbox(..., fontfile=str(fontdir / ttf_file)))
```

**Comment by JorjMcKie** [comment]
Of course - thank you.
Will be contained in coming v1.19.5.

**Comment by JorjMcKie** [comment]
Fixed in v1.19.5 currently being uploaded.
